soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -82,7 +82,6 @@ void z_intel_adsp_ipc_isr(const void *devarg)
 		}
```

Comment:
A side-note: While I find the detailed git commit description very useful as a SOF developer, I think it has a bit too much SOF/app specific knowledge (like SET_DX message) that will be hard to follow in context of Zephyr. Being too verbose is probably not too big of a problem (especially as most people working on this specific driver will use it with SOF), but as a general rule it would be good to describe the problem without using SOF/application terms, and maybe add the mapping to SOF in a separate paragraph in the commit message.